Remove value_changed call at init

RafaelLyra8 opened the pull request:
This was done because this function will always break, as the self.__value_changed_callback_ref will only be set in the poll function. This was discovered due to the log added by Marcus some weeks ago.

marcus-oscarsson commented:
Thanks, well done. Feel free to rebase and merge

fabcor-maxiv commented:
This probably could use some squashing of some sort. I do not think this should be two commits.

RafaelLyra8 committed:
Remove value_changed call at init

This was done because this function will always break, as the
self.__value_changed_callback_ref will only be set in the poll
function. This was discovered due to the log added by Marcus some weeks ago.
